services/datcore-adapter/src/simcore_service_datcore_adapter/core/application.py

```diff
@@ -22,24 +25,25 @@
 def create_app(settings: Optional[Settings] = None) -> FastAPI:
     if settings is None:
         settings = Settings.create_from_envs()
+    assert settings  # nosec
 
     logging.basicConfig(level=settings.LOG_LEVEL.value)
     logging.root.setLevel(settings.LOG_LEVEL.value)
+    for package in ["aiocache", "botocore", "hpack"]:
+        logging.getLogger(package).setLevel(logging.WARNING)
```

Comment:
i see some hacking here to get "noise" down :-) 
The only MINOR issue here is that if one day these libraries are not anymore there the loggers will not exists but you are unnecessarily creating them.

Reply:
I am using the same logic as you did in the webserver now. but I guess one way to not create loggers would be something like they describe [here](https://www.roelpeters.be/check-if-python-logger-already-exists/) but I'm not so sure whether it is really an issue...

Reply:
No, it is I said something minor!